src/shared/components/GlobalStyle/GlobalStyle.tsx

```diff
@@ -42,6 +42,17 @@ const globalStyles = css`
   }
 `
 
-const GlobalStyle: React.FC = () => <Global styles={globalStyles} />
+type GlobalStyleProps = {
+  additionalStyles?: SerializedStyles[] | SerializedStyles
+}
+
+const GlobalStyle: React.FC<GlobalStyleProps> = ({ additionalStyles }) => {
+  const additionalStylesArray = additionalStyles
```

Comment:
Could you elaborate on why this is necessary and we can't just include all styles into the same `css` fragment?
I feel like global styles should be pretty narrow in general and there should a be good reason to extend them

Reply:
Yeah, this is so we can add Atlas-specific global styles to the app as well. This component lives in `shared` so we shouldn't add them here. Otherwise we would need to add two separate global styles components in `LayoutWithRouting`. Does that make sense?

Reply:
Ah yes good point, let's keep it this way then.